libpod/filters/pods.go

```diff
@@ -78,7 +79,11 @@ func GeneratePodFilterFunc(filter, filterValue string) (
 		}, nil
 	case "name":
 		return func(p *libpod.Pod) bool {
-			return strings.Contains(p.Name(), filterValue)
+			match, err := regexp.MatchString(filterValue, p.Name())
+			if err != nil {
```

Comment:
I think we either return the error here or we at least warn log to the user that the provided regex does not compile. The new behavior probably needs documentation in one of the man pages, too.

Reply:
+1 to log a warning here

Reply:
@joelsmith could you update by adding a warning please and we'll move this along?

Reply:
Sure. Do you also want me to add a warning in the `podman ps` code path that I copied from? ( https://github.com/containers/podman/blob/master/libpod/filters/containers.go#L41 )

Reply:
@joelsmith  WHAT? Do you want consistency across the code?   Actually, that would be awesome, TYVM!  ;^)